packages/grid/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -58,6 +58,8 @@ type GridCellV7Props = {
 
 type GridCellWrapperProps = GridCellV7Props;
 
+type GridCellV7RootProps = DataGridProcessedProps & { unstable_cellSelection: boolean };
```

Comment:
Instead of updating this `type` I'd recommend using `@ts-expect-error` where it's necessary to use some prop from the `Pro` or `Premium` packages in `MIT` code.
Here's an example:
https://github.com/mui/mui-x/blob/34a9216cebceb805a4bbf684c709b4c25e49d5eb/packages/grid/x-data-grid/src/hooks/features/keyboardNavigation/useGridKeyboardNavigation.ts#L108